lib/src/Uncertainty/Algorithm/Optimization/LOLAVoronoi.cxx

```diff
@@ -84,11 +85,12 @@ String LOLAVoronoi::__repr__() const
 void LOLAVoronoi::computeVoronoiScore() const
 {
   // estimate the Voronoi cell size by Monte Carlo (2.5.1.1, algorithm 2)
-  const Sample voronoiSample(distribution_.getSample(voronoiSamplingSize_));
+  const UnsignedInteger voronoiSamplingSize = std::max(voronoiMinimumSamplingSize_, static_cast<UnsignedInteger>(voronoiSamplingRatio_ * x_.getSize()));
```

Comment:
voronoiMinimumSamplingSize_ is related to the global budget for the computation of the volumes, and voronoiSamplingRatio_ is in fact the mean number of points used to estimate the volume of a cell for uniformly distributed x_. So its name should reflect this, and also its type (it is an UnsignedInteger). IMO, a minimum value should be 100.

Reply:
done (using a default value of 10)